editor-support.md

```diff
@@ -20,10 +20,22 @@ Quicklisp, SLIME and Git.
 
 ### Installing SLIME
 
-SLIME is in the official GNU ELPA repository of Emacs Lisp packages
-(in Emacs24 and forward). Install with:
+On Ubuntu, SLIME is easily installed alongside Emacs and SBCL:
 
-    M-x package-install RET slime RET
+    sudo apt install emacs slime sbcl
+
+Otherwise, install SLIME by adding this code to your `~/.emacs` file:
```

Comment:
I think  ~/.emacs.d/init.el is preferred now.

Thanks again!

Reply:
Updated, thank you!